> mvn site is failing: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-4196/1/artifact/out/patch-mvnsite-root.txt
> 
> ```
> [ERROR] Failed to execute goal org.apache.maven.plugins:maven-site-plugin:3.11.0:site (default-site) on project hadoop-project: Execution default-site of goal org.apache.maven.plugins:maven-site-plugin:3.11.0:site failed: An API incompatibility was encountered while executing org.apache.maven.plugins:maven-site-plugin:3.11.0:site: java.lang.AbstractMethodError: org.apache.maven.doxia.module.markdown.MarkdownParser.parse(Ljava/io/Reader;Lorg/apache/maven/doxia/sink/Sink;Ljava/lang/String;)V
> ```
> 
> I think the error is caused by the doxia upgrades such as https://issues.apache.org/jira/browse/MSITE-878. Would you remove the hard-coded doxia version in pom.xml?
> 
> https://github.com/apache/hadoop/blob/de2e265ba0a8da2d84dd7f540871c8d14d1192a1/pom.xml#L316-L320
> 
> I removed the above lines and `mvn site` succeeded on my local.

@aajisaka  - I will do that.

--- a/pom.xml
+++ b/pom.xml
@@ -313,11 +313,6 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/x
               <artifactId>wagon-ssh</artifactId>
               <version>${wagon-ssh.version}</version>
             </dependency>
-            <dependency>
-              <groupId>org.apache.maven.doxia</groupId>
-              <artifactId>doxia-module-markdown</artifactId>
-              <version>1.8</version>
-            </dependency>
           </dependencies>
         </plugin>
         <!--This plugin's configuration is used to store Eclipse m2e settings only.

